src/elements/content-sidebar/SidebarNavSignButton.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classnames from 'classnames';
+import { injectIntl, IntlShape } from 'react-intl';
+import BoxSign28 from '../../icon/logo/BoxSign28';
+import Sign16 from '../../icon/fill/Sign16';
+import PlainButton, { PlainButtonProps } from '../../components/plain-button';
+// @ts-ignore Module is written in Flow
+import TargetedClickThroughGuideTooltip from '../../features/targeting/TargetedClickThroughGuideTooltip';
+import Tooltip, { TooltipPosition } from '../../components/tooltip';
+// @ts-ignore Module is written in Flow
+import messages from './messages';
+import './SidebarNavSignButton.scss';
+
+export type Props = PlainButtonProps & {
+    blockedReason?: string;
+    intl: IntlShape;
+    status?: string;
+    targetingApi?: {
+        canShow: boolean;
+        onClose: () => void;
+        onComplete: () => void;
+        onShow: () => void;
+    };
+};
+
+export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
+
+export function SidebarNavSignButton({ blockedReason, intl, status, targetingApi, ...rest }: Props) {
+    const isSignDisabled = !!blockedReason;
+    const isTargeted = targetingApi && targetingApi.canShow;
+    const FtuxTooltip = !isSignDisabled && isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
+    const label = intl.formatMessage(status === 'active' ? messages.boxSignSignature : messages.boxSignRequest);
+    const buttonClassName = classnames('bcs-SidebarNavSign', { 'bdl-is-disabled': isSignDisabled });
```

Comment:
since this file is updating several classnames (to follow SUIT naming), do we know if the classnames are also used directly in the parent app? (e.g. styling in a SCSS file)

Reply:
I checked it and the classnames aren't being used